tooling/template-base-essentials/components/CommunityNav.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const CommunityNav = () => {
+  const CommunityNavElement = (props: {
```

Comment:
Don't think it needs its own `CommunityNavElement`, does it? Could just loop through an array of objects again to render that.

Reply:
I have created a NavElementArray of objects, which is looped through in RenderNavElements to return the CommunityNav